docs/0006-flat-layout.md

```diff
@@ -0,0 +1,68 @@
+# OCFL Community Extension 0006: Flat OCFL Storage Layout
+
+  * **Extension Name:** 0006-flat-layout
+  * **Authors:** Peter Winckles
+  * **Minimum OCFL Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+## Overview
+
+This storage root extension describes a simple flat OCFL storage layout. OCFL object identifiers are mapped directly to directory names that are direct children of the OCFL storage root directory.
+
+This layout has significant limitations, and should never be used in a production repository. The specific limitations are filesystem dependent, but are generally as follows:
```

Comment:
Can you remove the first sentence? `This layout has significant limitations, and should never be used in a production repository.` This may not be the case for cloud storage.

Reply:
Fixed